csm/core/services/health.py

```diff
@@ -586,7 +587,9 @@ async def update_health_schema_with_db(self):
         if not self._is_map_updated_with_db:
             Log.debug(f"Updating health schema_with db.")
             try:
-                alerts = await self.alerts_repo.retrieve_by_range(create_time_range=None)
+                limits = QueryLimits(const.ES_RECORD_LIMIT, 0)
+                alerts = await self.alerts_repo.retrieve_by_range(create_time_range=None, limits=limits)
```

Comment:
is there any changes required for Auditlogs?

Reply:
No changes required.